app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt

```diff
@@ -121,6 +142,26 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 		 */
 		var ac3Enabled = booleanPreference("pref_bitstream_ac3", !DeviceUtils.isFireTvStickGen1)
 
+		/**
+		 * Enable EAC3
+		 */
+		var eac3Enabled = booleanPreference("pref_bitstream_eac3", !DeviceUtils.isFireTvStickGen1)
+
+		/**
+		 * Enable TrueUD
+		 */
+		var thdEnabled = booleanPreference("pref_bitstream_thd", false)
```

Comment:
I don't think we should shorten this to `thd`, doesn't seem like an "official" abbreviation.